index.js

```diff
@@ -310,6 +327,10 @@ const ContainerService = {
   },
 
   sortTabs() {
+    sendEvent({
```

Comment:
We are not going to have the same payload for each event, but we can have a method like this:

this._sendEvent(event, "totalTab", "foobar", "barfoo");

and sendEvent parses the arguments in order to create the correct payload.